src/main/java/com/scottlogic/filters/AuthorPostFilter.java

```diff
@@ -12,7 +12,7 @@
     private final String Author;
 
     public AuthorPostFilter(String authorInput) {
-        Author = authorInput;
+        this.Author = authorInput;
```

Comment:
A small point, the standard formatting for java might be something that is worth starting to implement now you have a better idea of the structure. Types and classes are normally `CapitalCamelCase`, variables and methods are normally `lowerCamelCase` even inside a class, so `author` would be preferred here and finally constants are often given `UPPERCASE_WITH_UNDERSCORE`. These are simply so any java programmer can come along and see things a bit easier, a slight warning this changes between languages a lot so `C#` has a completely different send of traditions for example